lib/ruby_indexer/lib/ruby_indexer/index.rb

```diff
@@ -199,6 +251,47 @@ class Class < Namespace
 
       class Constant < Entry
       end
+
+      class UnresolvedAlias < Entry
+        extend T::Sig
+
+        sig { returns(String) }
+        attr_reader :target
+
+        sig { returns(T::Array[String]) }
+        attr_reader :nesting
+
+        sig do
+          params(
+            target: String,
+            nesting: T::Array[String],
+            name: String,
+            file_path: String,
+            location: YARP::Location,
+            comments: T::Array[String],
+          ).void
+        end
+        def initialize(target, nesting, name, file_path, location, comments) # rubocop:disable Metrics/ParameterLists
```

Comment:
It's a lot of parameters, I wonder if we should use keywords instead? It may require some refactoring.

Reply:
Let's do the refactor in a different PR, since it will require quite a few changes.